src/main/java/com/google/api/codegen/metacode/InitCodeContext.java

```diff
@@ -82,6 +82,23 @@
         .collect(ImmutableList.toImmutableList());
   }
 
+  /** Returns a copy of this initCodeContext, but with an empty symbolTable. */
```

Comment:
What do you think about using AutoValue to generate a toBuilder method, and then just modifying the symbol table? (See https://github.com/google/auto/blob/master/value/userguide/builders-howto.md#-initialize-a-builder-to-the-same-property-values-as-an-existing-value-instance)

Reply:
That's a nice feature!